packages/suite/src/constants/suite/anchors.ts

```diff
@@ -35,6 +35,7 @@ export const enum SettingsAnchor {
     GithubIssue = '@debug-settings/github-issue',
     WipeData = '@debug-settings/wipe-data',
     InvityApi = '@debug-settings/invity-api',
+    OauthApi = '@debug-settings/oauth-api',
```

Comment:
```suggestion
    OAuthApi = '@debug-settings/oauth-api',
```

Reply:
Done in 469efbf93ea70aef646477c7b77bf76e1481be54